levels/management.tex

```diff
@@ -0,0 +1,234 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=525mm, paperheight=325mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills able.
+\newcolumntype{C}{p{20em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Engineering Managment
+\end{huge}
+\end{center}
+
+\bigbreak
+
+% Alternating row colors.
+\rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{center}
+\begin{tabular}{|C|C|C|C|C|}
+\hline
+    \rowcolor{blue!60!black!10}
+    \textbf{Level}
+    &
+    \cellcolor{blue!25}
+    \textbf{Product Output}
+    &
+    \textbf{Communication/Writing}
+    &
+    \textbf{Team Development}
+    &
+    \textbf{People Development}
+    \\
+\hline
+    % Level
+    Team Lead
+    &
+
+    % Product Output
+    \cellcolor{blue!25}
+    Team ships software reasonably on time with low regressions. Consistently
+    assess and front-loads risk.
+
+    \bigbreak
+
+    Team has low firefighting load.
+
+    \bigbreak
+
+    Proposes quarterly goals based on customer feedback.
```

Comment:
```suggestion
    Proposes quarterly goals based on customer feedback, platform performance, and operational toil.
```

Not sure how generic we want to make this. This addition is more relevant to cloud.